src/main/java/de/tum/in/www1/artemis/domain/ProgrammingExerciseTestCase.java

```diff
@@ -0,0 +1,141 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import javax.persistence.*;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/**
+ * A ProgrammingExerciseTestCase.
+ */
+@Entity
+@Table(name = "programming_exercise_test_case")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+public class ProgrammingExerciseTestCase implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "testName")
```

Comment:
```suggestion
    @Column(name = "test_name")
```

Reply:
You are right, it is more obvious.
A side not: This doesn't affect the mysql db, here it is already `test_name`, so we don't have to change anything on the servers.